internal/ci/ci.go

```diff
@@ -1,9 +1,16 @@
 package ci
 
-import "os"
+import (
+	"os"
+	"runtime"
+)
 
-// IsAvailable detects whether this code is executed inside a CI environment.
+// IsAvailable detects whether this code is executed inside a CI environment that isn't Windows or MacOS
```

Comment:
Imho, let's leave the win/mac logic out and just keep the `os.Getenv("SKIP_CI")` part.

Reply:
@alexplischke yeah I guess there is a chance that we'd need to run saucectl from inside a Windows or macOS VM.